src/providers/python.rs

```diff
@@ -136,6 +155,33 @@ enum EntryPoint {
 }
 
 impl PythonProvider {
+    fn is_django(app: &App, _env: &Environment) -> Result<bool> {
+        Ok(app.includes_file("manage.py"))
```

Comment:
I think this check should be a bit more detailed? Should it also check if `Django` is in the requirements file?